rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/BuildConfigurationEndpoint.java

```diff
@@ -317,8 +315,7 @@ BuildConfigurationRevision getRevision(
     Build triggerRevision(
             @Parameter(description = BC_ID) @PathParam("id") int id,
             @Parameter(description = REV) @PathParam("rev") int rev,
-            @BeanParam BuildParameters buildParams,
-            @Parameter(description = "Optional Callback URL") @QueryParam("callbackUrl") String callbackUrl);
+            @BeanParam BuildParameters buildParams);
```

Comment:
callback never really used in the old API eh?

Reply:
Yeah, and when it was used it was leaking internal password. We can add it back again, if we find out there really was some usecase.